.env.example

```
@@ -32,3 +32,13 @@ S3_SECRET_KEY=
 # Leave blank for AWS; set for B2 or other S3-compatible providers, e.g.:
 # S3_ENDPOINT=https://s3.us-west-004.backblazeb2.com
 S3_ENDPOINT=
+
+# ── Database backups ──────────────────────────────────────────────────────────
+# Daily pg_dump → s3://$S3_BUCKET/backups/ via the `db-backup` service.
+# Backups older than BACKUP_KEEP_DAYS are pruned after each successful upload.
+# Schedule is a standard crontab expression in UTC.
+BACKUP_SCHEDULE=0 3 * * *
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)